src/utils/string/index.ts

```diff
@@ -41,7 +41,6 @@ export const msToDuration = (ms: number): string => {
   const minutes = Math.floor(hoursMilli / (60 * 1000));
```

Comment:
(comment placement is random) maybe let's change the description element on the public keys preference page because it looks super super small 😅

<img width="357" alt="Screen Shot 2022-10-27 at 3 50 54 PM" src="https://user-images.githubusercontent.com/47064971/198386682-05c906a9-0663-4d6f-afe3-eee14f4d0c35.png">